locale/de.json

```diff
@@ -29,29 +29,29 @@
-            "deny_message": "Please join the guild %{guildname} then try again."
+            "mode_title": "Dieser Server ist im <strong>Discord Guild Member Whitelist</strong> Modus.",
+            "insufficient_ids": "Du hast keine <code>Discord</code> identifier, welche aber erforderlich sind, um zu überprüfen, ob du auf dem Discord-Server bist.",
+            "deny_title": "Du musst auf unserem Discord-Server sein, um dich zu verbinden",
+            "deny_message": "Bitte trete unserem Discord %{guildname} bei, dann versuche es erneut."
         },
         "guild_roles": {
-            "mode_title": "This server is in <strong>Discord Role Whitelist</strong> mode.",
-            "insufficient_ids": "You do not have the <code>discord</code> identifier, which is required to validate if you have joined our Discord Guild. Please open Discord and try again.",
-            "deny_notmember_title": "You are required to join our Discord Guild to connect.",
-            "deny_notmember_message": "Please join %{guildname}, get one of the required roles, then try again.",
-            "deny_noroles_title": "You do not have a whitelisted role required to join.",
-            "deny_noroles_message": "To join this server you are required to have at least one of the whitelisted roles on the guild %{guildname}."
+            "mode_title": "Dieser Server ist im <strong>Discord Rollen Whitelist</strong> Modus.",
+            "insufficient_ids": "Du hast keine <code>Discord</code> identifier, welche aber erforderlich sind. Um zu überprüfen, ob du auf dem Discord-Server bist, öffne Discord und versuche es erneut.",
+            "deny_notmember_title": "Du musst auf unserem Discord-Server sein, um dich zu verbinden",
+            "deny_notmember_message": "Bitte trete unserem Discord %{guildname} bei, hol dir eine der erforderlichen Rollen, dann versuche es erneut.",
+            "deny_noroles_title": "Du hast keine Whitelist rolle, die für das Verbinden erforderlich ist.",
+            "deny_noroles_message": "Um diesem Server beizutreten ist mindestens eine Whitelist Rolle benötigt, Discord: %{guildname}."
         },
         "approved_license": {
-            "mode_title": "This server is in <strong>License Whitelist</strong> mode.",
-            "insufficient_ids": "You do not have the <code>license</code> identifier, which means the server has <code>sv_lan</code> enabled. If you are the server owner, you can disable it in the <code>server.cfg</code> file.",
-            "deny_title": "You are not whitelisted to join this server.",
-            "request_id_label": "Request ID"
+            "mode_title": "Dieser Server ist im <strong>License Whitelist</strong> Modus.",
+            "insufficient_ids": "Du hast keine <code>License</code> identifier, was bedeutet dieser Server hat <code>sv_lan</code> aktiviert, bist du der Serverbesitzer, dann kannst du dies in der server.cfg ändern.",
```

Comment:
`"Du hast keine <code>License</code> identifier, was bedeutet dieser Server hat <code>sv_lan</code> aktiviert. Solltest du der Serverbesitzer sein, dann kannst du dies in der server.cfg ändern."`